src/config/partners/showtime.ts

```diff
@@ -15,14 +15,102 @@ const showtime: Partner = {
   twitter: '@Showtime_xyz',
   drops: [
     {
```

Comment:
@ximxim I think we're missing the headliner here - the 'Showtime Summer: Unlock Artists' Perks' entry